changelogs/api-management-service.md

```diff
@@ -1,5 +1,59 @@
 # Azure API Management service changelog
 
+# Release -API Management service: October 2023
+
+### Highlights
+
+1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2.(https://learn.microsoft.com/en-us/azure/api-management/v2-service-tiers-overview)
+
+### New features, fixes and improvements
+
+1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.
```

Comment:
```suggestion
### New features

None.

### Fixes and improvements

1. We have added a feature flag for exporting API security when exporting customer API to swagger.
```

Reply:
Aligned as per discussion, but is this a customer-facing feature flag or not? If not, then we should remove it

Reply:
@tomkerkhove, _swagger_ is a proper noun and should be capitalized. It's also capitalized [here](https://swagger.io/solutions/getting-started-with-oas/).